### PR TITLE
Add loadgen to deployment integration test

### PR DESCRIPTION
**Pull request by mhofman**
refs: #3444 

## Description

This change leverages the loadgen work, and stacks it on top of https://github.com/Agoric/agoric-sdk/pull/4133 to run a short loadgen cycle against the 2 node testnet started by the integration test.

Unlike #3107, the goal is not to generate perf data (even though some very basic deploy times might be useful), but to make sure changes to `agoric-sdk` do not break the loadgen which is a simple representative of dapps.

It should also help catch some non-determinism source (see #3444), but only if caused by different SwingSet executions. It would probably not catch restart related non-determinism.

### Security Considerations

This should not have any impact on security, and is modeled against the dapp integration test which also checks out another Agoric owned repository.

### Testing Considerations

In general SDK breaking changes should be accommodated in the loadgen in a backwards compatible manner, so that the loadgen can run against both old and new versions of the SDK. That usually means merging a loadgen fix first, then running the integration test in the SDK's PR. If necessary, an SDK PR can target a specific loadgen branch using the following PR notation: ` #loadgen-branch: main ` (replace `main` with the branch name).

### CODE REVIEW

File: packages/deployment/src/init.js
Comment (michaelfig) [inline packages/deployment/src/init.js:133]: I had to think twice about why this phrasing was necessary.  Please add a comment.
```suggestion
      // Tolerate NaN returned by any parseInt errors.
      if (!(cgroupVersion >= 2)) {
```

**Review by michaelfig — APPROVED** [review]
The approach and current details look great!

If minor tweaks to pass CI are all you need, you can consider this to be my approval.  Rerequest a review only if you make other changes.

**Comment by mhofman** [comment]
@michaelfig I have not materially changed the PR since the review, so I will merge in a few unless I get any objections (after removing the PR trigger)

**Pull request merged**
